portality/view/doaj.py

```diff
@@ -342,9 +342,11 @@ def toc(identifier=None):
 
 
 @blueprint.route("/toc/articles/<identifier>")
-@blueprint.route("/toc/articles/<identifier>/<volume>")
-@blueprint.route("/toc/articles/<identifier>/<volume>/<issue>")
-def toc_articles(identifier=None, volume=None, issue=None):
+def toc_articles_legacy(identifier=None):
+    return redirect(url_for('doaj.toc_articles', identifier=identifier, volume=volume, issue=issue), 301)
```

Comment:
The call to `redirect()` won't work, since volume and issue aren't provided.

Reply:
Good point. Fixed.